### PR TITLE
Unified `created` event

### LINKED ISSUE #253: `kytos.mef_eline.created` is being sent twice during EVC creation
`kytos.mef_eline.created` is being sent twice during EVC creation:

https://github.com/kytos-ng/mef_eline/blob/master/main.py#L236-L239

https://github.com/kytos-ng/mef_eline/blob/master/main.py#L244

This event should only be sent once on creation. The new `telemetry_int` NApp will subscribe to it. Let's send the entire evc dict representation, `telemetry_int` NApp will only need to derive the cookie and metadata at this moment, but having the rest of the attributes can be helpful for logging or any other new functionality since it won't cost much to send it all let's send it. 

@Alopalao, could you help out with this one as well since your working on related issues? Thanks.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #253 

### Summary
- Deleted first `created` event from EVC creation since it was being sent twice.
- Changed `emit_event()` arggument from `**kwargs` to `content=None` to avoid conflicts with `name`

### Local Tests
- Changed `test_handle_link_down()` to work with recent changes.
- Changed every `emit_event()` call to suit new argument

### CODE REVIEW

**Review by Ktmi — CHANGES_REQUESTED** [review]

File: utils.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 from kytos.core.events import KytosEvent
 
 
-def emit_event(controller, name, context="kytos/mef_eline", **kwargs):
+def emit_event(controller, _name, context="kytos/mef_eline", **kwargs):
```
[the remark this answers is not in the document]
Comment (Alopalao) [reply utils.py:17]: Got it.

[the remark this answers is not in the document]
Comment (Alopalao) [reply utils.py:17]: Could it be like this?: `def emit_event(controller, name, context="kytos/mef_eline", extra={}, **kwargs)` 
And the event could be created like:  `KytosEvent(name=event_name, content=kwargs|extra)`
`kwargs` flexibility is already being use by more functions.

[the remark this answers is not in the document]
Comment (Ktmi) [reply utils.py:17]: You could do that, but the issue is that it makes reading code like this confusing:
```
emit_event(
    self.controller,
    context="kytos.flow_manager",
    name="flows.install",
    dpid=dpid,
    flow_dict={"flows": switch_flows[dpid][:offset]},
)
```
The point were the content starts, and the other arguments ends isn't clear. By comparison, wrapping the content in a dict is far clearer:

```
emit_event(
    self.controller,
    context="kytos.flow_manager",
    name="flows.install",
    content={
        'dpid`: dpid,
        'flow_dict': {"flows": switch_flows[dpid][:offset]},
    }
)
```

[the remark this answers is not in the document]
Comment (Alopalao) [reply utils.py:17]: Ok

[the remark this answers is not in the document]
Comment (Alopalao) [reply utils.py]: Changed on commit [010e7c06b1ec4f5347f96b4d03dab11f07d6346b](https://github.com/kytos-ng/mef_eline/pull/255/commits/010e7c06b1ec4f5347f96b4d03dab11f07d6346b)

**Review by Ktmi — APPROVED** [review]

**Review by italovalcy — APPROVED** [review]
LGTM! Thanks for the PR, Aldo!

**Review by viniarck — APPROVED** [review]
Nicely done, @Alopalao. :rocket:

**Pull request merged**